File: cpp/p10.cpp
#include <iostream>

bool isPrime(int num)
{
    if (num <= 1)
    {
        return false;
    }

    for (int i = 2; i * i <= num; ++i)
    {
        if (num % i == 0)
        {
            return false;
        }
    }

    return true;
}

int main(int argc, char const *argv[])
{
    long long ans = 0;
    for (size_t i = 0; i < 2000000; i++)
    {
        if (isPrime(i))
        {
            ans += i;
        }
    }

    std::cout << ans << std::endl;

    return 0;
}

File: python/p10.py
def is_prime(number):
    if number < 2:
        return False

    for i in range(2, int(number ** 0.5) + 1):
        if number % i == 0:
            return False
        
    return True

def main():
    ans = 0
    for i in range(1, 2000000):
        if is_prime(i):
           ans += i

    print(ans)

if __name__ == "__main__":
    main()